e2e-tests/native/test/helpers/Gestures.ts

```diff
@@ -0,0 +1,178 @@
+import { RectReturn } from '@wdio/protocols/build/types'
```

Comment:
Checkout the original https://github.com/webdriverio/appium-boilerplate/blob/main/tests/helpers/Gestures.ts